machinery/topology.go

```diff
@@ -186,6 +186,10 @@ func (t *Topology) ToDot() string {
 	return t.graph.String()
 }
 
+func (t *Topology) Graph() *dot.Graph {
```

Comment:
Is this func needed for the upgrade?

Reply:
No, it isn't. It will unblock some exploration into exposing more info from the graph elsewhere however (and looked like a useful thing to expose to machinery users).

Reply:
Yeah that sounds good - the rest of the dot lib does similar with things that return pointer to edges, nodes or (now) the graph

Reply:
I added deep copy to `dot` - it has been released `v1.8.0`

Will update this to use

Reply:
done.